_episodes/05d-singletons.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Other Data Types"
+teaching: 5
+exercises: 0
+questions:
+- "What are True, False, and None?"
+objectives:
+- "Learn why True, False, and None are different from other datatypes"
+keypoints:
+- "`True`, `False`, and `None` are special keywords in Python"
+- "There is only ever one object called `True`, `False`, or `None`"
+---
+
+# True, False, and None are special data types
```

Comment:
True and False should be talked about using the word "Boolean". They are also their own datatype (bool) which is different from None (which is no data type). When introducing bools, it may also be beneficial to talk about the int values of bools (1 or 0).